packages/sdk-react/src/mocks/reminders/remindersHandlers.ts

```diff
@@ -0,0 +1,174 @@
+import { paths } from '@/api';
+
+import { delay, http, HttpResponse } from 'msw';
+
+import {
+  createOverdueReminderListItemFixture,
+  createPaymentReminderListItemFixture,
+  overdueReminderListFixture,
+  paymentReminderListFixture,
+} from './reminderListFixtures';
+
+export const remindersHandlers = [
+  http.post<
+    {},
+    paths['/payment_reminders']['post']['requestBody']['content']['application/json'],
+    paths['/payment_reminders']['post']['responses']['201']['content']['application/json']
+  >('*/payment_reminders', async ({ request }) => {
+    const jsonBody = await request.json();
+
+    const newPaymentReminder = createPaymentReminderListItemFixture(jsonBody);
+    paymentReminderListFixture.push(newPaymentReminder);
+
+    await delay();
+    return HttpResponse.json(newPaymentReminder);
+  }),
+
+  http.post<
+    {},
+    paths['/overdue_reminders']['post']['requestBody']['content']['application/json'],
+    paths['/overdue_reminders']['post']['responses']['201']['content']['application/json']
+  >('*/overdue_reminders', async ({ request }) => {
+    const jsonBody = await request.json();
+
+    const newOverdueReminder = createOverdueReminderListItemFixture(jsonBody);
+    overdueReminderListFixture.push(newOverdueReminder);
+
+    await delay();
+    return HttpResponse.json(newOverdueReminder);
+  }),
+
+  http.patch<
+    { payment_reminder_id: string },
+    | paths['/payment_reminders/{payment_reminder_id}']['patch']['requestBody']['content']['application/json'],
+    | paths['/payment_reminders/{payment_reminder_id}']['patch']['responses']['200']['content']['application/json']
+    | paths['/payment_reminders/{payment_reminder_id}']['patch']['responses']['404']['content']['application/json']
+  >('*/payment_reminders/:payment_reminder_id', async ({ request, params }) => {
```

Comment:
Are this huge types:
```ts
    | paths['/payment_reminders/{payment_reminder_id}']['patch']['requestBody']['content']['application/json'],
    | paths['/payment_reminders/{payment_reminder_id}']['patch']['responses']['200']['content']['application/json']
    | paths['/payment_reminders/{payment_reminder_id}']['patch']['responses']['404']['content']['application/json']
```
 to just cast `*/payment_reminders/:payment_reminder_id'` to follow open api spec?